### PR TITLE
TEST PR PLZ IGNORE

### PR DESCRIPTION
**Pull request by rynowak**
TEST PR PLZ IGNORE

### CODE REVIEW

**Comment by glennc** [comment]
No.

File: README.md [hunk 1]
```diff
@@ -1,10 +1 @@
...
-===
-
-AppVeyor: [![AppVeyor](https://ci.appveyor.com/api/projects/status/github/aspnet/HttpClientFactory?branch=dev&svg=true)](https://ci.appveyor.com/project/aspnetci/HttpClientFactory/branch/dev)
-
-Travis:   [![Travis](https://travis-ci.org/aspnet/HttpClientFactory.svg?branch=dev)](https://travis-ci.org/aspnet/HttpClientFactory)
-
-Contains an opinionated factory for creating HttpClient instances.
-
-This project is part of ASP.NET Core. You can find samples, documentation and getting started instructions for ASP.NET Core at the [Home](https://github.com/aspnet/home) repo.
+TEST PR PLZ IGNORE
```
Comment (muratg) [inline README.md]: NIT: PLZ -> PLEAZE

**Comment by rynowak** [comment]
@Eilon assures me that appveyor is working, yet, execute it does not.

**Comment by Eilon** [comment]
It's weird because it's building the repo, but not PRs...

**Comment by Eilon** [comment]
THERE it is!

**Comment by Eilon** [comment]
I deleted & recreated the project on AppVeyor. It looks happy now.

**Comment by rynowak** [comment]
Will merge once @muratg is happy

**Pull request closed without merging**